core/internal/runfiles/uploader.go

```diff
@@ -301,6 +301,8 @@ func (u *uploader) upload(relativePaths []string) {
 		return
 	}
 
+	u.logger.Info("runfiles: uploading files", "files", relativePaths)
```

Comment:
will it be too loud if we have many files? should we have it as debug?
maybe it's fine?

Reply:
fair point, changed to Debug